components/components/blocks/search/icon-search.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+    <title>noun_932673_00A388</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Front-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Umami---Front---Sketch-1---Desktop" transform="translate(-120.000000, -19.000000)" fill-rule="nonzero" fill="#00A388">
+            <g id="noun_932673_00A388" transform="translate(120.000000, 19.000000)">
+                <path d="M17.8354167,13.6416667 C20.7490131,10.432074 20.6298848,5.50045427 17.5647153,2.43528471 C14.4995457,-0.629884848 9.56792597,-0.749013062 6.35833333,2.16458333 C3.73288804,4.75188633 3.22773033,8.80349255 5.1375,11.95625 C5.2186819,12.0897563 5.19895166,12.2613236 5.08958333,12.3729167 L0.95,16.50625 C0.116666667,17.3395833 -0.0708333333,18.4833333 0.660416667,19.2145833 L0.785416667,19.3395833 C1.51666667,20.0708333 2.66041667,19.875 3.49375,19.05 L7.625,14.91875 C7.73523737,14.8080329 7.90601696,14.7849776 8.04166667,14.8625 C11.1948998,16.773384 15.247789,16.2681739 17.8354167,13.6416667 Z M7.85625,12.14375 C6.14138854,10.4289784 5.62835291,7.85003893 6.55637558,5.609511 C7.48439825,3.36898307 9.67071653,1.90811119 12.0958333,1.90811119 C14.5209501,1.90811119 16.7072684,3.36898307 17.6352911,5.609511 C18.5633138,7.85003893 18.0502781,10.4289784 16.3354167,12.14375 C13.9920238,14.4805294 10.1996429,14.4805294 7.85625,12.14375 L7.85625,12.14375 Z" id="Shape"></path>
+                <path d="M8.375,7.42083333 C8.09151799,7.42386924 7.82593755,7.28257734 7.67004485,7.04578898 C7.51415216,6.80900061 7.48933361,6.50919999 7.60416667,6.25 C8.7445221,3.57390189 11.8348104,2.32455711 14.5145833,3.45625 C14.7903477,3.57273346 14.9830213,3.82748717 15.0200259,4.12454787 C15.0570304,4.42160856 14.932744,4.71584566 14.6939842,4.89642288 C14.4552244,5.0770001 14.1382644,5.11648348 13.8625,5 C12.0324647,4.22728679 9.92230882,5.08197446 9.14583333,6.91041667 C9.01547248,7.22055216 8.7114177,7.42188573 8.375,7.42083333 L8.375,7.42083333 Z" id="Shape"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
I would pass this SVG through a compressor to have cleaner code: https://jakearchibald.github.io/svgomg/. It actually removes a 54% of code, like Sketch comments. :)

Reply:
@ckrina as we sad in Slack ththis will be a followup issue: #76